tests/unit_tests/tools/test_compiler_wrapper.py

```diff
@@ -47,48 +45,10 @@ def test_version_and_caching(stub_c_compiler: CCompiler,
     # Test that the value is cached:
     assert mpicc.get_version() == (1, 2, 3)
     assert call_list(fake_process) == [
-        ['scc', '--version'],
         ['mpicc', '--version']
     ]
 
 
-def test_version_consistency(stub_c_compiler: CCompiler,
```

Comment:
I take it this test is no longer relevant. Are these the teste mentioned in the comment trail? If not a quick explanation of why they are removed would be handy.

Reply:
There is the comment `Removed test if wrapped compiler is consistent with the wrapper compiler.`, and I believe there is a quite long explanation in the corresponding ticket #407